src/App.tsx

```diff
@@ -1,156 +1,94 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { TodoForm } from './components/TodoForm/TodoForm';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { TodoNotification } from './components/TodoNotification/TodoNotification';
+import * as postServes from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { FilterType } from './types/enum';
+import { getPreperadTodos } from './utils/helper';
 
-const USER_ID = 0;
+export const USER_ID = 11122;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[] | null>([]);
+  // const [loading, setLoading] = useState<boolean>(false);
+  const [isError, setIsError] = useState<boolean>(false);
+  const [errorMessage, setErrorMessage] = useState<string>('');
```

Comment:
`isError` can be based on `errorMessage`
```suggestion
  const [errorMessage, setErrorMessage] = useState<string>('');
  const isError = !!errorMessage;
```